File: C++/Lista extra 02/1.c
#include <stdio.h>

int main(){
    float n1,n2;

    printf("Informe dois numeros:\n");
    scanf("%f",&n1);
    scanf("%f",&n2);

    if(n1>n2){
        printf("%f e maior que %f", n1,n2);    
    }
    if(n2>n1){
        printf("%f e maior que %f", n2,n1);    
    }   
    if(n2==n1){
        printf("%f = %f", n1,n2);    
    }

    return 0;
}

File: PYTHON/LISTA 2/1.py
#Faça um Programa que peça dois números e imprima o maior deles
numero1 = float(input("Informe o primeiro numero: "))
numero2 = float(input("Informe o segundo numero: "))

if numero1 > numero2:
    print("{} é maior que {}".format(numero1,numero2))
    
if numero1 < numero2:
    print("{} é maior que {}".format(numero2,numero1))
    
if numero1 == numero2:
    print("Os numeros são iguais")  
